static/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "TUM Mensa",
+  "short_name": "TUM Mensa",
+  "icons": [
+    {
+      "src": "/images/favicon-192.png",
```

Comment:
This path is wrong: It should be `/static` and there is no favicon of size `192x192` (yet).